stdlib/public/TensorFlow/Dataset.swift

```diff
@@ -185,14 +185,27 @@ extension DatasetIterator : IteratorProtocol {
   }
 }
 
+/// A 2-tuple-like struct that conforms to TensorGroup that represents the 
+/// result type of a zip operation between two datasets.
```

Comment:
This API is more general than datasets and thus should not mention "dataset" in its documentation.

Reply:
I’ll edit this to reflect your comment.